docs/Limitations.md

```diff
@@ -20,6 +20,10 @@ You can control the frequency of Academy stepping by calling
 `Academy.Instance.DisableAutomaticStepping()`, and then calling
 `Academy.Instance.EnvironmentStep()`
 
+### Unity Inference Engine Models
+Currently, it is only possible to use models created with our trainers to run
```

Comment:
Maybe something like "Currently, only models created with our trainers are supported for running ML-Agents with a neural network behavior."? It's _possible_ to make your own, but we can't help you :P

Reply:
I don't think we should say it is possible